client/src/components/navigation/Navbar.js

```diff
@@ -122,8 +122,8 @@ const Navbar = props => {
 
   const drawer = (
     <List component="nav" className={classes.list}>
-      <ListItemLink to="/refreshrs/create" className={classes.navRoutes} />
       <ListItemLink to="/classes/create" className={classes.navRoutes} />
```

Comment:
I'm only seeing `Create` on the Navbar, not `Create Classes`. Not sure if that's coming from here or somewhere else.

![image](https://user-images.githubusercontent.com/13543632/54007908-2ff0ef00-4132-11e9-82af-92e46e640c09.png)

Reply:
That's quite strange! Ya fix one thing and another breaks lol

Reply:
Yep, seems to be how it goes :laughing: